Archive.Zip.UnitTests/packages.config

```diff
@@ -1,12 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Castle.Core" version="4.4.0" targetFramework="net45" />
+  <package id="Moq" version="4.13.1" targetFramework="net45" />
   <package id="NUnit" version="2.6.2" targetFramework="net45" />
   <package id="Remotion.Development.TempFile.Sources" version="1.15.33.0" targetFramework="net45" developmentDependency="true" />
   <package id="Remotion.JetBrains.Annotations.Sources" version="1.15.23.0" targetFramework="net45" developmentDependency="true" />
   <package id="Remotion.Utilities.ArgumentUtility.Sources" version="1.15.23.0" targetFramework="net45" developmentDependency="true" />
   <package id="Remotion.Utilities.CultureScope.Sources" version="1.17.0" targetFramework="net45" />
   <package id="Remotion.Utilities.FileUtility.Sources" version="1.15.33.0" targetFramework="net45" developmentDependency="true" />
   <package id="Remotion.Utilities.NullableTypeUtility.Sources" version="1.15.23.0" targetFramework="net45" developmentDependency="true" />
-  <package id="RhinoMocks" version="3.6.1" targetFramework="net45" />
   <package id="SharpZipLib" version="0.86.0" targetFramework="net45" />
+  <package id="System.Runtime.CompilerServices.Unsafe" version="4.5.0" targetFramework="net45" />
```

Comment:
Oh dear, what is that? Is that actually Moq's dependency?

Reply:
Removed without breaking functionality.

Reply:
Redo please. All dependencies set by NuGet need to remain, otherwise the next update will complain and there might also be issues when restoring packages, etc.